types/types.go

```diff
@@ -48,6 +48,10 @@ type OrgID = types.OrgID
 // ImpactingFlag controls the behaviour of 'impacting' param on GET /rule/
 type ImpactingFlag int
 
+// RequestID is used to store the request ID supplied in input Kafka records as
+// a unique identifier of payloads. Empty string represents a missing request ID.
+type RequestID string
```

Comment:
probably later it should go to `types` as it is used in Aggregator too (and in IO operator as well).